lib/flipper/dsl.rb

```diff
@@ -34,6 +34,15 @@ def enabled?(name, *args)
       feature(name).enabled?(*args)
     end
 
+    def enabled_for_some?(name, actor)
```

Comment:
My colleague made a good point, that this is mostly about "enabled for actors" or actor-only feature flags that aren't enabled by percentage or globally. That's a much better way to put it. There may be a better way to model this deeper in the abstractions, but my hot take would be name this method `strict_enabled?`.